script/install/debootstrap.bash

```diff
@@ -171,5 +171,47 @@ echo "$chroot_cmd update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9
 chroot "$ISOLATE_ROOT" update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-9 75
 # gcc 9 done
 
-# let user know that chroot installs are finished
-
+# JavaScript V8 engine compile and install
+echo "Preparing to compile and install the JavaScript V8 engine."
+echo
+echo "If this fails, visit https://v8.dev/docs/build and follow the instructions."
+echo "This may take a while ..."
+echo
+set -x
+
+: Making temporary directory
+mkdir ~/v8-tools
+cd ~/v8-tools
+
+: Installing depot_tools
+git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git
+export PATH="$PATH":`pwd`/depot_tools
+gclient
+
+: Getting V8 source code
+mkdir v8
+cd v8
+fetch v8
+cd v8
+
+: Installing additional build dependencies
+gclient sync
+./build/install-build-deps.sh
+
+: Compiling V8
+./tools/dev/v8gen.py x64.release -- v8_use_external_startup_data=false
+ninja -C out.gn/x64.release d8
+
+: Moving executable into place
+mv ./out.gn/x64.release/d8 "$ISOLATE_ROOT"/usr/bin/d8
```

Comment:
Actually, I take it back -- I don't think copying `d8` instead of moving it is going to help much. On the other hand, something that I think _will_ help is if the script exited on errors (`set -e`) instead of continuing. Otherwise if something goes wrong (e.g. build error, or failure to copy `d8`) the script will still delete the entire directory, so everything would have to be downloaded from scratch.